src/classrooms/classrooms.controller.ts

```diff
@@ -41,4 +41,14 @@ export class ClassroomsController {
     const classroom = await this.classroomsService.getClassroomById(parseInt(id, 10), user.id);
     return this.classroomsSerializer.serialize(classroom);
   }
+
+  @Delete(":id")
+  @Roles(EUserType.TEACHER)
+  async deleteClassroom(
+    @Param("id") id: string,
+    @CurrentUser() user: { id: number },
+  ): Promise<boolean> {
+    const result = await this.classroomsService.deleteClassroom(parseInt(id, 10), user.id);
```

Comment:
```suggestion
    const result = await this.classroomsService.deleteClassroom(id, user.id);
```



Can't we just do this?

Reply:
Yes we can. Previously I had some unexpected issue that's why I did this.